addons/stock_dropshipping/models/purchase.py

```diff
@@ -2,6 +2,25 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 from odoo import api, models, fields
+from odoo.exceptions import UserError
+
+
+class SaleOrderLine(models.Model):
+    _inherit = 'sale.order.line'
+
+    @api.multi
+    def write(self, values):
+        if 'product_uom_qty' in values:
+            self.ensure_one()
```

Comment:
Maybe iterate over the lines that bulk update of ```product_uom_qty``` is made possible even if I think this is never used in core and not very smart from a functional point of view.